vuepress/docs/.vuepress/next.js

```diff
@@ -3,67 +3,115 @@ module.exports = {
         return [
             {
                 title: 'Introduction',
-                path: path,
-            },
-            {
-                title: 'Getting Started',
                 children: [
                     {
                         title: 'Overview',
-                        path: path  + 'getting-started/',
+                        path: path,
                     },
                     {
-                        title: 'Welcome Wizard',
-                        path: path + 'getting-started/welcome-wizard.md'
+                        title: 'Getting Started',
+                        path: path + 'getting-started/',
+                    },
+                    {
+                        title: 'Concepts',
+                        path: path + 'getting-started/concepts-overview.md',
                     },
                 ]
             },
             {
-                title: 'Concepts',
+                title: 'Create Components',
                 children: [
                     {
-                        title: 'Overview',
-                        path: path + 'concepts/'
+                        title: 'Entando Component Generator',
+                        path: path + 'create/component-gen-overview.md',
                     },
                     {
-                        title: 'Entando Operator',
-                        path: path + 'concepts/operator-intro.md'
+                        title: 'Entando Blueprint Features',
+                        path: path + 'create/blueprint-features.md',
                     },
                     {
-                        title: 'Custom Resources',
-                        path: path + 'concepts/custom-resources.md'
+                        title: 'Component Generation Technologies',
+                        path: path + 'create/component-gen-tech.md',
                     },
                     {
-                        title: 'Accessibility',
-                        path: path + 'concepts/accessibility.md'
+                        title: 'Customizing The Component Generator',
+                        path: path + 'create/component-gen-customize.md',
+                    },
+                ]
+            },
+            {
+                title: 'Curate and Share Components',
+                children: [    
+                    {
```

Comment:
this link has to be removed, otherwise the expand/contract won't work. I think it has to do with the fact that entando-hub.md (not hub.md) is in the tutorial directory

Reply:
sorry! i had changed hub.md to entando-hub.md - both tutorial file names + the references to them in the js files

Reply:
no problem, it's not the name of the file that's at issue anyways. I don't think we can cross from docs to tutorials in the sidebar.

Reply:
Let's discuss on the call. That was one I was unclear about.